### ISSUE TITLE
[Focus] Pastes from clipboard on every keystroke [iOS 14 Support] 

### ISSUE THREAD

**Issue by athomasmoz** [mozilla-mobile/firefox-ios#6932]
https://github.com/mozilla-mobile/focus-ios/issues/1711

**Comment by nbhasin2** [comment]
Apple actually has a new iOS14 api just to check if its a URL for paste and go. Although it needs more investigation on how to properly use it.
```
static let number: UIPasteboard.DetectionPattern
static let probableWebSearch: UIPasteboard.DetectionPattern
static let probableWebURL: UIPasteboard.DetectionPattern
```
https://developer.apple.com/documentation/uikit/uipasteboard/detectionpattern?changes=latest_minor

**Comment by nbhasin2** [comment]
https://developer.apple.com/forums/thread/649996

**Comment by SimonBasca** [comment]
Still reproducible on Focus 8.1.4 on iOS 14b2. 
STR
1. Copy a link from Safari
2. Open Focus 
3. Tap on the URL bar

**Comment by nbhasin2** [comment]
[Screen Recording 2020-07-14 at 10.22.02 AM.mov.zip](https://github.com/mozilla-mobile/firefox-ios/files/4919337/Screen.Recording.2020-07-14.at.10.22.02.AM.mov.zip)

@SimonBasca If you paste then yes it should show the top notification but not if you simply tap.

I am on master branch and fixed it over there. 

Could you look at the video and let me know if the steps I am doing are correct as I am unable to reproduce the issue on Master branch?

**Comment by SimonBasca** [comment]
Still reproducible on latest bdb0c51e5dd7 on sim iPhone 11 Pro. 

STR

1. Copy a link from Safar
2. Open Focus
3. Tap on the URL bar
4. Start typing in the URL bar

[Screen Recording 2020-07-16 at 12.22.26 PM.mov.zip](https://github.com/mozilla-mobile/firefox-ios/files/4930556/Screen.Recording.2020-07-16.at.12.22.26.PM.mov.zip)

**Comment by nbhasin2** [comment]
@SimonBasca its because the PR is still up due to other test failure. I am rebuilding so I can merge https://github.com/mozilla-mobile/focus-ios/pull/1712 then you can retest again. 

Moving it back in development for now.

**Comment by nbhasin2** [comment]
https://github.com/mozilla-mobile/focus-ios/pull/1712 - Merged 
Moving it to done. cc @SimonBasca

**Comment by SimonBasca** [comment]
Verified fix on latest master 294e9e3346549c on sim iPhone 11 (14.0).
